test/system/reports_test.rb

```diff
@@ -50,15 +50,14 @@ class ReportsTest < ApplicationSystemTestCase
   end
 
   test 'should destroy Report' do
-    visit report_url(@report)
-
+    visit reports_url
     assert_text '初めての日報'
-    assert_text '緊張しました'
 
+    visit report_url(@report)
     click_on 'この日報を削除'
 
+    assert_selector 'h1', text: '日報の一覧'
     assert_text '日報が削除されました。'
     assert_no_text '初めての日報'
```

Comment:
```suggestion
    assert_current_path reports_url
    assert_no_text '初めての日報'
```

ちゃんと元の一覧ページに戻っていることを確認するとなおよし、です

参考
https://qiita.com/jnchito/items/1eea181854038929f16f#actiondispatchsystemtestcase-applicationsystemtestcase-%E3%81%A7%E8%BF%BD%E5%8A%A0%E3%81%95%E3%82%8C%E3%82%8B%E6%A4%9C%E8%A8%BC%E3%83%A1%E3%82%BD%E3%83%83%E3%83%89